localstack-core/localstack/services/events/utils.py

```diff
@@ -273,3 +273,7 @@ def is_nested_in_string(template, match) -> bool:
         return False
 
     return left_quote != -1 and template[left_quote + 1 : right_quote].strip() != match.group(0)
+
+
+def dict_to_simple_string(d):
```

Comment:
Ah, I missed this one too: I think this does not cover all use case if there are nested values here. 
It seems what AWS does is stripped all quotes from the JSON string, it might be easier that way.

Because what if you have a nested dict values here? I don't think that works, same if `v` is a list with strings inside.

Reply:
I added a test case for nested dicts in string transformer `'{"method": "PUT", "nested": {"level1": {"level2": {"level3": "users-service/users/<userId>"} } }, "bod": "<userId>"}',`